test/plotting.jl

```diff
@@ -0,0 +1,30 @@
+@testset "ZonedDateTime plot recipe" begin
+    start_zdt = ZonedDateTime(2017,1,1,0,0,0, tz"EST")
+    end_zdt = ZonedDateTime(2017,1,1,10,30,0, tz"EST")
+    zoned_dates = start_zdt:Hour(1):end_zdt
+
+    expected_dates = DateTime.(zoned_dates, Local)
+
+    @testset "No label" begin
+        result, = RecipesBase.apply_recipe(Dict{Symbol, Any}(), zoned_dates, 1:11)
+        xs, ys = result.args
+        @test xs == expected_dates
+        @test ys == 1:11
+        @test result.plotattributes[:xguide] == "(timezone: EST)"
+    end
+
+    @testset "label (should append to it)" begin
+        result, = RecipesBase.apply_recipe(Dict{Symbol, Any}(:xguide=>"Hi"), zoned_dates, 1:11)
+        xs, ys = result.args
+        @test xs == expected_dates
+        @test ys == 1:11
+        @test result.plotattributes[:xguide] == "Hi (timezone: EST)"
+    end
+
+    @testset "No items" begin
+        empty_xs = ZonedDateTime[]
+        empty_ys = 0:-1
+        result = RecipesBase.apply_recipe(Dict{Symbol, Any}(:xguide=>"Hi"), empty_xs, empty_ys)
+        @test true  # we are just making sure it it didn't error
```

Comment:
We can test that the `:xguide` is unmodified

Reply:
We can't actually, because this does not actually return the usual thing.
Because the recipe bails out early because this is an invalid senario.